### PR TITLE
chore: format and standardize all markdowns files

### PR DESCRIPTION
**Pull request by storopoli**
according to the github flavor
(https://github.github.com/gfm/)

### CODE REVIEW

File: bitcoin/tests/data/serde/README.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 
 Files here contain hex strings and binary data representing types used for
 regression testing.
 
-- *_hex: consensus encoded types represented as hex strings
-- *_ser: consensus encoded types represented as binary data
-- *_bincode: types serialized with serde as bincode
+- `*_hex`: consensus encoded types represented as hex strings
+- `*_ser`: consensus encoded types represented as binary data
+- `*_bincode`: types serialized with `serde` as bincode
```
Comment (storopoli) [inline bitcoin/tests/data/serde/README.md:8]: Opinionated: this should be highlighted as code

File: bitcoin/tests/data/serde/README.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 
-We consensus deserialize, serde serialize, then check against the expected data
-to verify no serde regressions have been introduced.
+We consensus deserialize, `serde` serialize, then check against the expected data
+to verify no `serde` regressions have been introduced.
```
Comment (storopoli) [inline bitcoin/tests/data/serde/README.md:11]: We already highlight "serde" as inline code, so this is standardizing it.

File: fuzz/README.md [hunk 3]
```diff
@@ -16,11 +18,13 @@ recently-released binutils 2.39 has changed their API in a breaking way.
...
-    nix-shell -p libopcodes_2_38 -p libunwind
+```bash
+nix-shell -p libopcodes_2_38 -p libunwind
+```
 
-and then run fuzz.sh as above.
+and then run `fuzz.sh` as above.
 
-# Fuzzing with weak cryptography
+## Fuzzing with weak cryptography
```
Comment (storopoli) [inline fuzz/README.md:27]: A markdown file should have only one H1.

File: hashes/extended_tests/schemars/README.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ Run as usual with `cargo test`.
...
 To run the tests with the MSRV you will need to pin `serde`:
 
-- `cargo update -p serde --precise 1.0.156`
-- `cargo update -p regex --precise 1.7.3`
-- `cargo update -p chrono --precise 0.4.24`
+```bash
+cargo update -p serde --precise 1.0.156
+cargo update -p regex --precise 1.7.3
+cargo update -p chrono --precise 0.4.24
+```
```
Comment (storopoli) [inline hashes/extended_tests/schemars/README.md:13]: Opinionated: this should be a fenced block for nice syntax highlighting

File: io/README.md [hunk 5]
```diff
@@ -1,9 +1,8 @@
-Rust-Bitcoin IO Library
-=======================
+# Rust-Bitcoin IO Library
 
 The `std::io` module is not exposed in `no-std` Rust so building `no-std` applications which require
 reading and writing objects via standard traits is not generally possible. Thus, this library exists
-to export a minmal version of `std::io`'s traits which we use in `rust-bitcoin` so that we can
+to export a minimal version of `std::io`'s traits which we use in `rust-bitcoin` so that we can
```
Comment (storopoli) [inline io/README.md:5]: typo

File: logo/README.md [hunk 6]
```diff
@@ -23,7 +23,7 @@ Rust Logo is licensed under CC-BY, which allows reuse and modifications for any
 
 ## Acknowledgements
 
-Acknowledgement for the runners up in this PR: https://github.com/rust-bitcoin/rust-bitcoin/pull/891#issuecomment-1074476858
+Acknowledgement for the runners up in this PR: <https://github.com/rust-bitcoin/rust-bitcoin/pull/891#issuecomment-1074476858>
```
Comment (storopoli) [inline logo/README.md:26]: Automatic formatted by my linter.
We can revert if necessary.

**Review by apoelstra — APPROVED** [review]
ACK b355740da43233958990998bb53b4d6e19069c06 thanks for going through all this!

**Review by tcharding — APPROVED** [review]
Sweeet, thanks man!

ACK b355740da43233958990998bb53b4d6e19069c06

**Pull request merged**